### ISSUE TITLE
Non-functional icons/spacers should be excluded from assistive technology

### ISSUE THREAD

**Issue by jeromehardaway** [Vets-Who-Code/vets-who-code-app#357]
"Requirement: Font icons, SVG or images that are being used as spacers, decorations or their purpose is already described by the content should include a "role" attribute that equals to "presentation" or "none"."

On the about page give the font stars the role of "presentation"
